crates/bevy_window/src/lib.rs

```diff
@@ -4,16 +4,18 @@ mod window;
 mod windows;
 
 use bevy_ecs::system::IntoSystem;
+use bevy_math::Vec2;
 pub use event::*;
+use std::collections::HashMap;
```

Comment:
I think you should use `bevy_utils::HashMap` instead. It is configured for maximum performance and recommended for most things in bevy (and your own bevy apps!).

Reply:
Ahh, great, will do! :)